src/test/java/org/testdata/TestData.java

```diff
@@ -0,0 +1,20 @@
+package org.testdata;
+
+import org.example.Man;
+import org.example.Woman;
+
+public class TestData {
+    //adding men
+    protected Man man1 = new Man("Kelsey", "Gardner", 43, null);
+    protected Man man2 = new Man("Christopher", "Glisson", 89, null);
+    protected Man man3 = new Man("Winston", "Cox", 65, null);
+    protected Man man4 = new Man("Mark", "Brown", 30, null);
+    protected Man man5 = new Man("Dwayne", "Preston", 30, null);
```

Comment:
For testing retaired I sugeest to use boundary testing. Age 64, 65, 66 for men and 59, 60, 61 for woman to check that retaired is working fine